### PR TITLE
Remove some unused or EOLed plugins

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 1]
```diff
@@ -30,7 +30,6 @@
     'blueocean-rest-impl'                => {'version' => '1.13.2'},
     'blueocean-web'                      => {'version' => '1.13.2'},
     'branch-api'                         => {'version' => '2.1.2'},
-    'checkstyle'                         => {'version' => '4.0.0'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: deprecated for warnings-ng

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 2]
```diff
@@ -41,7 +40,6 @@
     'credentials-binding'                => {'version' => '1.18'},
     'dashboard-view'                     => {'version' => '2.10'},
     'display-url-api'                    => {'version' => '2.3.0'},
-    'dry'                                => {'version' => '3.0.0'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: deprecated for warnings-ng

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 3]
```diff
@@ -96,7 +93,6 @@
     'timestamper'                        => {'version' => '1.9'},
     'token-macro'                        => {'version' => '2.7'},
     'variant'                            => {'version' => '1.2'},
-    'warnings'                           => {'version' => '5.0.1'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: deprecated for warnings-ng

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 4]
```diff
@@ -78,7 +76,6 @@
     'workflow-scm-step'                  => {'version' => '2.7'},
     'workflow-step-api'                  => {'version' => '2.19'},
     'workflow-support'                   => {'version' => '3.2'},
-    'xunit'                              => {'version' => '2.3.3'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: not used afaict

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 5]
```diff
@@ -122,7 +118,6 @@
     'authentication-tokens'              => {'version' => '1.3'},
     'blueocean-display-url'              => {'version' => '2.2.0'},
     'bouncycastle-api'                   => {'version' => '2.17'},
-    'brakeman'                           => {'version' => '0.12'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: not used afaict

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 6]
```diff
@@ -149,7 +144,6 @@
     'icon-shim'                          => {'version' => '2.0.3'},
     'instant-messaging'                  => {'version' => '1.35'},
     'ircbot'                             => {'version' => '2.30'},
-    'jclouds-jenkins'                    => {'version' => '2.14'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: not used afaict

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 7]
```diff
@@ -163,7 +157,6 @@
     'parameterized-trigger'              => {'version' => '2.35.2'},
     'pipeline-github-lib'                => {'version' => '1.0'},
     'pipeline-githubnotify-step'         => {'version' => '1.0.4'},
-    'xvfb'                               => {'version' => '1.1.3'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: should have been removed in https://github.com/theforeman/foreman-infra/commit/5e241b7eff2ab6d522ad0fae2cd64031d07f8ad7 along with the changes in the two other files

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 8]
```diff
@@ -173,8 +166,6 @@
     'pubsub-light'                       => {'version' => '1.12'},
     'pxe'                                => {'version' => '1.5'},
     'rake'                               => {'version' => '1.8.0'},
-    'redmine'                            => {'version' => '0.21'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: not used afaict

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 9]
```diff
@@ -173,8 +166,6 @@
...
-    'regression-report-plugin'           => {'version' => '1.5'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: not used afacit

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 10]
```diff
@@ -186,8 +177,6 @@
     'ssh'                                => {'version' => '2.6.1'},
     'statusmonitor'                      => {'version' => '1.3'},
     'throttle-concurrents'               => {'version' => '2.0.1'},
-    'translation'                        => {'version' => '1.16'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: no longer works

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 11]
```diff
@@ -186,8 +177,6 @@
...
-    'violations'                         => {'version' => '0.7.11'},
```
Comment (tbrisker) [inline puppet/modules/jenkins_master/manifests/init.pp]: not used afaict

**Comment by mmoll** [comment]
are these plugins for sure unused in the current jobs?

**Comment by tbrisker** [comment]
I couldn't find places they are used, but i may have missed some.

**Comment by mmoll** [comment]
Allright, then I'd say, we should try it and deal with the fallout later.

**Comment by ekohl** [comment]
In https://github.com/theforeman/foreman-infra/pull/979 I've added the Vagrantfile I used locally to test. Not sure if we should be adding that to our infra repo, but I thought it was good to share.

**Comment by tbrisker** [comment]
closing for #997

**Pull request closed without merging**